.github/workflows/lint.yaml

```diff
@@ -38,4 +38,5 @@ jobs:
 
     - name: Run Lint
       run: npm run lint
```

Comment:
```
- name: Run eslint
   run: npm run lnit
-name: Run solhint
   run: npm run lint:sol
   ```
 this way we can have separate sections on the report and it will be more easy for debugging.